source/features/select-notifications.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+import {
+	CheckCircleIcon,
+	CheckIcon,
+	DotFillIcon,
+	DotIcon,
+	GitMergeIcon,
+	GitPullRequestDraftIcon,
+	GitPullRequestIcon,
+	IssueOpenedIcon,
+	XCircleIcon
+} from '@primer/octicons-react';
+
+import features from '.';
+
+const filters = {
+	'Pull requests': ':is(.octicon-git-pull-request, .octicon-git-pull-request-closed, .octicon-git-pull-request-draft, .octicon-git-merge)',
+	Issues: ':is(.octicon-issue-opened, .octicon-issue-closed)',
+	Open: ':is(.octicon-issue-opened, .octicon-git-pull-request)',
+	Closed: ':is(.octicon-issue-closed, .octicon-git-pull-request-closed)',
+	Draft: '.octicon-git-pull-request-draft',
+	Merged: '.octicon-git-merge',
+	Read: '.notification-read',
+	Unread: '.notification-unread'
+};
+
+type Filter = keyof typeof filters;
+type Category = 'Type' | 'Status' | 'Read';
+
+function resetFilters({target}: Event): void {
+	for (const checkbox of select.all('input[type="checkbox"]:checked', target as Element)) {
+		checkbox.checked = false;
+		checkbox.parentElement!.parentElement!.setAttribute('aria-checked', 'false');
+	}
```

Comment:
Is there a "native" way to do this? I think that the GitHub component would try to keep these 2 attributes in sync. Maybe calling `form.reset()`?

Reply:
> I think that the GitHub component would try to keep these 2 attributes in sync

Looking at the custom element's code, this only happens when a checkbox is checked/unchecked. I think it's faster to call `reset()` on the form and reset all the `aria-checked` attributes.